frontends/web/src/locales/en/app.json

```
@@ -197,6 +197,40 @@
   "bitboxBase": {
     "connect": "Connect Base",
     "detectedBases": "Detected the following devices on your network:",
+    "errors": {
+      "BACKUP_SYSCONFIG_NOT_A_MOUNTPOINT": "Internal error: {{error}}",
+      "CHANGEPASSWORD_TOO_SHORT": "Password too short. A password must be at least 8 characters.",
+      "CMD_SCRIPT_INVALID_ARG": "Internal error: {{error}}",
+      "CONFIG_SCRIPT_INVALID_ARG": "Internal error: {{error}}",
+      "DUMMY_AUTHENTICATION_NOT_SUCCESSFUL": "Internal error: {{error}}",
+      "DUMMY_CHANGEPASSWORD_TOO_SHORT": "Internal error: {{error}}",
+      "ENABLE_CLEARNETIBD_TOR_ALREADY_DISABLED": "Tor is disabled. Cannot set Bitcoin Core to use Tor with public internet blockchain download.",
+      "EXECUTABLE_NOT_FOUND": "Internal error: {{error}}",
+      "FLASHDRIVE_CHECK_MULTI": "More than one flashdrive plugged in to the BitBoxBase, please remove all but one.",
+      "FLASHDRIVE_CHECK_NONE": "No flashdrive plugged in to the BitBoxBase, please insert one.",
+      "FLASHDRIVE_MOUNT_NOT_FOUND": "Internal error: {{error}}",
+      "FLASHDRIVE_MOUNT_NOT_SUPPORTED": "Flashdrive plugged in to the BitBoxBase is not supported (either too big or filesystem not supported).",
+      "FLASHDRIVE_MOUNT_NOT_UNIQUE": "Internal error: {{error}}",
+      "FLASHDRIVE_UNMOUNT_NOT_MOUNTED": "Internal error: {{error}}",
+      "MENDER_UPDATE_ALREADY_IN_PROGRESS": "Cannot update firmware, update is already in progress.",
+      "MENDER_UPDATE_COMMIT_FAILED": "Internal error: {{error}}",
```

This could do with a user facing error message as well, or? You decide.

Something along the lines of:
```suggestion
      "MENDER_UPDATE_COMMIT_FAILED": "Could not finalize the BitBoxBase update.",
```

---

Changed error msg to "Could not finalize the BitBoxBase update (commit failed)."